tensorboard/webapp/runs/views/runs_table/runs_data_table.ts

```diff
@@ -40,6 +42,7 @@ export class RunsDataTable {
   @Input() regexFilter!: string;
   @Input() isFullScreen!: boolean;
   @Input() selectableColumns!: ColumnHeader[];
+  @Input() columnFilters!: Map<string, DiscreteFilter | IntervalFilter>;
```

Comment:
This is only being passed down to for use in the filter dialogue and that feels strange to me for some reason but I can't place my finger on why.

Reply:
Seems fine to me.